src/util/findAndMoveHelpers.js

```diff
@@ -124,8 +113,12 @@ const moveAnnotation = function(
     tool.name,
     annotation,
     null,
-    toolOptions,
-    interactionType
+    tool.options,
+    interactionType,
+    () => {
```

Comment:
Looks like `doneMovingCallback` was pulled out and added as a param. This feels like something to aggregate in migration guide notes?

Reply:
Yeah, its documented on the project board :)